bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -1474,25 +1479,41 @@ template <class R, class... Args>
 void BcStTest::getMissingblocksStage(std::function<R(Args...)> callAtStart,
```

Comment:
We need to rename and refactor skipReplyOnce/rejectOnce and think how to make it clearer. Although the logic works, the usability and clarity of this function is not good. It is not clear what will happen on each combination of the pair. From the logic, you allow to skip and to reject at the same time but i dont think it is desired by caller as well.